src/adagrams.js

```diff
@@ -1,8 +1,75 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+const _ = require('lodash');
+class Adagrams {
+
+
+  static drawLetters() {
+    const letters = [];
+    const letter_freq = new Map([
+      ['A', 9], ['N', 6], ['B', 2], ['O', 8], ['C', 2], ['P', 2], ['D', 4], 
+      ['Q', 1], ['E', 12], ['R', 6], ['F', 2], ['S', 4], ['G', 3], ['T', 6], 
+      ['H', 2], ['U', 4], ['I', 9], ['V', 2], ['J',1], ['W', 2], ['K', 1], 
+      ['X', 1], ['L', 4], ['Y',2], ['M', 2], ['Z', 1]]);
+    for (let [key, value] of letter_freq) {
+        for (let i = 0; i < value; i++) {
+              letters.push(key);
+        }  
```

Comment:
I really like that you store the letter frequencies and then build the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.